Source/Core/DiscIO/DiscScrubber.cpp

```diff
@@ -59,7 +56,7 @@ void DiscScrubber::MarkAsUsed(u64 offset, u64 size)
   u64 current_offset = Common::AlignDown(offset, CLUSTER_SIZE);
   const u64 end_offset = offset + size;
 
-  DEBUG_LOG(DISCIO, "Marking 0x%016" PRIx64 " - 0x%016" PRIx64 " as used", offset, end_offset);
+  DEBUG_LOG_FMT(DISCIO, "Marking 0x{:016x} - 0x{:016x} as used", offset, end_offset);
```

Comment:
I've seen this being used in other places, so I'll just ask: would it make sense to use the extended format modifier here instead of putting the `0x` prefix in the message?
```suggestion
  DEBUG_LOG_FMT(DISCIO, "Marking {:#018x} - {:#018x} as used", offset, end_offset);
```